docs/usage.md

```diff
@@ -68,22 +68,27 @@ on this issue to provide some recommendations.
 Sometimes VCF files will contain the
 ancestral state in the "AA" ("ancestral allele") info field, in which case it will be encoded
 in the `variant_AA` field of the .vcz file. It's also possible to provide a numpy array
-of ancestral alleles, of the same length as the number of variants. Ancestral
-alleles that are not in the list of alleles for their respective site are treated as unknown
-and not used for inference (with a warning given).
+of ancestral alleles, of the same length as the number of selected variants. If you have a string
+of the ancestral states (e.g. from FASTA) the {meth}`add_ancestral_state_array`
+method can be used to convert and save this to the VCF Zarr dataset (under the name
+`ancestral_state`). Note that the positions are treated as 1-based.
+Alleles that are not in the list of alleles for their respective site are
+treated as unknown and not used for inference (with a warning given).
 
 ```{code-cell}
 import tsinfer
+import zarr
 
-# For this example take the REF allele (index 0) as ancestral
-ancestral_state = ds['variant_allele'][:,0].astype(str)
-# This is just a numpy array, set the last site to an unknown value, for demo purposes
-ancestral_state[-1] = "."
-
-vdata = tsinfer.VariantData("_static/example_data.vcz", ancestral_state)
+vcf_zarr = zarr.open("_static/example_data.vcz")
+ancestral_string = "GGCTCAG."
```

Comment:
What I find simplest is to prepend (say) "X" to the FASTA string so that it's effectively in one-based indexes. I think we need to help people out a bit more here as they'll definitely get it wrong.